### PR TITLE
update scripts and docker file

### PR DESCRIPTION
**Pull request by Nargeshdb**

### CODE REVIEW

File: fse-2021/LICENSE.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
-Our tool itself is distributed under an MIT license. This license also applies
-to our scripts, which were originally part of the tool repository.
+# Released under MIT License
```
Comment (kelloggm) [inline fse-2021/LICENSE.md:1]: I don't think this set of changes is correct.

I suggest retaining the original text, and then adding the license text underneath it. It's important to make clear that the case study programs are not released under MIT, which would be a violation of their license terms. (I also think the reviewer who complained about the license was being obtuse, but I don't object to including the actual license terms after the text that makes it clear to what it applies.)

File: fse-2021/LICENSE.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
 
-Our modifications to the case study programs (and those programs themselves)
-are distributed under their respective open-source licenses.
\ No newline at end of file
+Copyright (c) 2021 Martin Kellogg
```
Comment (kelloggm) [inline fse-2021/LICENSE.md:3]: All of our names should appear here, not just mine.

**Review by kelloggm — CHANGES_REQUESTED** [review]

File: fse-2021/LICENSE.md (hunk 1 above)
Comment (Nargeshdb) [reply fse-2021/LICENSE.md]: https://github.com/kelloggm/object-construction-checker/pull/471/commits/48c83052952914d857105be55e76926598875eb1

**Review by msridhar — APPROVED** [review]

**Comment by msridhar** [comment]
LGTM, @kelloggm want to take another look?

**Review by kelloggm — APPROVED** [review]

**Pull request merged**